src/xpring-client.ts

```diff
@@ -146,68 +149,84 @@ class XpringClient {
     return this.getFee().then(async fee => {
       return this.getAccountInfo(sender.getAddress()).then(
         async accountInfo => {
-          if (accountInfo.getSequence() == undefined) {
-            return Promise.reject(
-              new Error(XpringClientErrorMessages.malformedResponse)
-            );
-          }
-
-          const xrpAmount = new XRPAmount();
-          xrpAmount.setDrops(normalizedAmount.toString());
-
-          const payment = new Payment();
-          payment.setXrpAmount(xrpAmount);
-          payment.setDestination(destination);
-
-          const transaction = new Transaction();
-          transaction.setAccount(sender.getAddress());
-          transaction.setFee(fee);
-          transaction.setSequence(accountInfo.getSequence());
-          transaction.setPayment(payment);
-          transaction.setSigningPublicKeyHex(sender.getPublicKey());
-
-          var signedTransaction;
-          try {
-            signedTransaction = Signer.signTransaction(transaction, sender);
-          } catch (signingError) {
-            const signingErrorMessage =
-              XpringClientErrorMessages.signingFailure +
-              ". " +
-              signingError.message;
-            return Promise.reject(new Error(signingErrorMessage));
-          }
-          if (signedTransaction == undefined) {
-            return Promise.reject(
-              new Error(XpringClientErrorMessages.signingFailure)
-            );
-          }
-
-          const submitSignedTransactionRequest = new SubmitSignedTransactionRequest();
-          submitSignedTransactionRequest.setSignedTransaction(
-            signedTransaction
-          );
+          return this.getLastValidatedLedgerSequence().then(
+            async ledgerSequence => {
+              if (accountInfo.getSequence() == undefined) {
```

Comment:
`== undefined` checks if the getSequence() value is `null or undefined` if you want to strictly check `undefined` then use `===`

Reply:
Done